projects/ion/src/lib/dropdown/dropdown.component.html

```diff
@@ -5,6 +5,12 @@
   (clickOutside)="clickedOutsideDropdown()"
   [class.container-options--loading]="loading"
 >
+  <span
+    *ngIf="description && description.length > 0"
```

Comment:
```suggestion
    *ngIf="description"
```
Acredito que você posso validar usando apenas `description` ou `!!description`.